x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/otel_kubernetes/otel_kubernetes_panel.tsx

```diff
@@ -218,7 +219,7 @@ kind: Pod
 metadata:
   name: my-app
   annotations:
-    instrumentation.opentelemetry.io/inject-${idSelected}: "true"
+    instrumentation.opentelemetry.io/inject-${idSelected}: "${namespace}/elastic-instrumentation"
```

Comment:
Seems like this change also needs to be made for the CopyToClipboardButton - that's still saying `"true"`

Reply:
Good catch!